src/v2/Apps/Conversation/Components/ConfirmArtworkModal.tsx

```diff
@@ -59,10 +61,12 @@ export const ConfirmArtworkModal: React.FC<ConfirmArtworkModalProps> = ({
             disabled={!!isEdition && !selectedEdition}
             conversationID={conversationID}
             editionSetID={selectedEdition || null}
-            trackingEvent={{
-              context_module: OwnerType.conversationMakeOfferConfirmArtwork,
-              context_owner_type: OwnerType.conversation,
-            }}
+            onClick={() =>
+              tracking.trackEvent({
+                context_module: OwnerType.conversationMakeOfferConfirmArtwork,
+                context_owner_type: OwnerType.conversation,
+              })
+            }
```

Comment:
Why was this moved from the child component if we don't have to get any data from the parent? 🤔

Reply:
I guess because of the way we use it, we use it from `MakeOfferOnInquiryButton` which dispatches a different event than this modal here

Reply:
The child component `ConfirmArtworkButton` needs to track different events depending on the context (confirm artwork, click purchase, click make offer).
In this case, the ConfirmArtworkModal is tracking the "confirm artwork" event. So, it's not because of the data it needs, but because of who triggers what. Thoughts?